Contribution Agreement.md

```diff
@@ -0,0 +1,24 @@
+# Contributor Agreement
+
+All contributors to ChakraCore must digitally sign the below agreement.
+
+Anything I contribute to the ChakraCore repository is comprised of one or more of:
+
```

Comment:
Two thoughts on reflection on this:
1. Do we need a clause to cover work done in the course of employment if any company ever contributes anything (it's unlikely they will but it's possible and it would be awkward to have to adapt this agreement at that time)
2. Do we need an explicit exception for anything microsoft submits up to March - they had stated they were going to do security fixes until then

Reply:
I've updated the document to possibly permit this - though would appreciate second opinions.